src/ui/public/courier/data_source/_doc_send_to_es.js

```diff
@@ -32,8 +32,8 @@ export default function (Promise, Private, es) {
     if (validateVersion && params.id) {
       params.version = doc._getVersion();
     }
-
-    return es[method](params)
+    const client = _.includes(params.index, kbnIndex) ? esAdmin : es;
```

Comment:
At this point in the code it's hard to say if it's string or array. Both support includes but with different equality conditions. Maybe `[].concat(params.index).includes(kbnIndex)`